runtime/queries/cairo/locals.scm

```diff
@@ -16,53 +16,51 @@
   (if_expression)
   (match_expression)
   (match_arm)
-
   (struct_item)
   (enum_item)
   (impl_item)
 ] @local.scope
 
-(use_declaration 
-  argument: (scoped_identifier 
-              name: (identifier) @local.definition.import))
+(use_declaration
+  argument:
+    (scoped_identifier
+      name: (identifier) @local.definition.import))
 
-(use_as_clause 
+(use_as_clause
   alias: (identifier) @local.definition.import)
 
 (use_list
-    (identifier) @local.definition.import) ; use std::process::{Child, Command, Stdio};
+  (identifier) @local.definition.import) ; use std::process::{Child, Command, Stdio};
 
 ; Functions
-
 (function_definition
-    (identifier) @local.definition.function)
+  (identifier) @local.definition.function)
 
 (function_definition
   (identifier) @local.definition.method
-  (parameter (self)))
+  (parameter
+    (self)))
 
 ; Function with parameters, defines parameters
-
 (parameter
-  [ (identifier) (self) ]  @local.definition.parameter)
+  [
+    (identifier)
+    (self)
+  ] @local.definition.parameter)
 
 ; Types
-
 (struct_item
   name: (type_identifier) @local.definition.type)
 
-(constrained_type_parameter 
+(constrained_type_parameter
   left: (type_identifier) @local.definition.type) ; the P in  remove_file<P: AsRef<Path>>(path: P)
```

Comment:
```suggestion
 ; the P in  remove_file<P: AsRef<Path>>(path: P)
(constrained_type_parameter
  left: (type_identifier) @local.definition.type)
```

Reply:
Inline comments must be conserved; manual changes should only be made in this PR if they are required for the auto-formatter to work properly.